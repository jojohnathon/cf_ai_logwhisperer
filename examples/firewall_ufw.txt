Jun 21 12:10:01 host kernel: [UFW BLOCK] IN=eth0 OUT= MAC=ff:ff:ff:ff:ff:ff SRC=192.168.1.50 DST=224.0.0.251 LEN=78 TOS=0x00 PREC=0x00 TTL=1 ID=54321 PROTO=UDP SPT=5353 DPT=5353 LEN=58
Jun 21 12:10:05 host dhclient[2001]: DHCPDECLINE on eth0 to 192.168.1.1 port 67
Jun 21 12:10:07 host dhclient[2001]: DHCPDECLINE on eth0 to 192.168.1.1 port 67
