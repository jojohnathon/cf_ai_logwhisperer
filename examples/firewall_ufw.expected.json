{
  "summary": "UFW is blocking mDNS multicast while DHCP lease negotiations are failing.",
  "anomalies": [
    "Repeated DHCPDECLINE events",
    "UFW dropping UDP/5353 multicast"
  ],
  "suggested_commands": [
    { "cmd": "sudo ufw allow 5353/udp", "why": "Allow mDNS multicast traffic", "risk": "low" },
    { "cmd": "sudo systemctl restart NetworkManager", "why": "Reset DHCP client state", "risk": "med" }
  ]
}
